draft-ietf-quic-multipath.md

```diff
@@ -1358,16 +1358,20 @@ MAX_PATH_ID frames that do not increase the path limit MUST be ignored.
 A sender SHOULD send a PATHS_BLOCKED frame (type=0x15228c0d) when
 it wishes to open a path but is unable to do so due to the maximum path identifier
 limit set by its peer.
+If the Type field in the PATHS_BLOCKED frame is TBD-09(experiments use 0x15228c0e),
+it indicates that new paths are blocked because of lacking unused Connection IDs
+for the corresponding Path ID.
+
 Note that PATHS_BLOCKED frame is informational. Sending a PATHS_BLOCKED frame does not
 imply a particular action from the peer like updating the new Max Path ID value,
-but informs the peer that the maximum path identifier limit prevented the creation of new paths.
-
+but informs the peer that the maximum path identifier limit prevented the creation of new paths
+or the lacking of new Connection IDs.
```

Comment:
I don't think "lacking" works here grammatically, either use "lack of Connection IDs for any unused path ID".